src/sgame/sg_bot_util.cpp

```diff
@@ -1675,33 +1675,38 @@ void BotFireWeapon( weaponMode_t mode, usercmd_t *botCmdBuffer )
 }
 void BotClassMovement( gentity_t *self, bool inAttackRange )
 {
-	usercmd_t *botCmdBuffer = &self->botMind->cmdBuffer;
+	botMemory_t *mind = self->botMind;
+	usercmd_t *botCmdBuffer = &mind->cmdBuffer;
+	bool botIsSmall = false;
+	bool botIsJumper = false;
+
+	// If we are targetting a buildable or we are targetting a human who
+	// isn't looking at us (likely fleeing), so we try to reach the human
+	// as fast as we can.
+	// Making special tricks for buildables is still useful because turrets
+	// are slow to aim and defenders do more team-damage this way.
+	bool shouldStrafe = mind->goal.getTargetType() != entityType_t::ET_PLAYER
```

Comment:
The meaning of this variable seems to be should approach directly i.e. *shouldn't* strafe?

Also it seems wrong to use Alignment2D which is made for direction vectors on `angles`.

Reply:
I mean ultimately, what is currently named Alignment2D is just that, an angle. Except it's actually the cosine of an angle. If you want I can rename it Angle2D, putting a warning that you should use acosf(Angle2D(…)) if you actually want the angle

Reply:
No, `angles` is like Euler angles, not a vector. Comparing it with a dot product is not even approximately correct since you have 0 degrees vs. 359 degrees etc.

Reply:
Well, here you have an angle from 0 to 180°. But ok for keeping the name.

The Alignment2D lines are equivalent if that's what you were afraid of.

About the strafe thing. Do you mean that the condition is inverted (wrong) or that it would be more readable inverted?

Reply:
LGTM